api/tests/unit/features/versioning/test_unit_versioning_views.py

```diff
@@ -194,6 +194,44 @@ def test_publish_feature_version(
     )
 
 
+@pytest.mark.parametrize("live_from", (None, tomorrow))
+def test_publish_feature_version_using_master_api_key(
+    admin_master_api_key_client: "APIClient",
+    environment_v2_versioning: Environment,
+    feature: "Feature",
+    live_from: typing.Optional[datetime],
```

Comment:
This is minor, but given that this is a test module shouldn't we be able to do the following?
```suggestion
    admin_master_api_key_client: APIClient,
    environment_v2_versioning: Environment,
    feature: Feature,
    live_from: datetime | None,
```

Reply:
Yes, fair point, I can do that.

Reply:
Done (although I didn't update the rest of the quoted uses in the test module as I didn't want to dilute this PR). I think it's worth doing that in a separate PR.

Reply:
Scrap that, I've updated the rest of the file here too.